.github/workflows/prod.yaml

```diff
@@ -9,7 +9,7 @@ jobs:
     uses: Cerebellum-Network/reusable-workflows/.github/workflows/build-and-upload-static.yaml@1.0.0
     with:
       runs-on: '["self-hosted", "cere-network-large-workers"]'
-      build_container: 'node:14-buster'
+      build_container: 'node:lts-buster'
```

Comment:
Variable versioning may introduce non-deterministic builds. Is it intentional or we may use something like`node:18-buster` here instead?

Reply:
Thank you for your comment!
1. LTS stands for "last stable version" and is usually updated once or twice a year.
2. The Polkadot-JS team has built all of their codebase and GitHub workflow for the LTS version. For example, all pipeline versions have the `lts` node version.